src/main/steps/applicant1/check-your-conditional-order-answers/content.ts

```diff
@@ -16,12 +16,20 @@ const en = ({ isJointApplication, isDivorce, userCase }: CommonContent) => ({
     isInformationCorrect: 'Is the information in this application still correct?',
   },
   stepAnswers: {
-    continueApplication: userCase.applicant1ApplyForConditionalOrder,
-    isInformationCorrect: userCase.applicant1ConfirmInformationStillCorrect,
+    continueApplication: `${
```

Comment:
how comes here its continueApplication and actually property names and in check your answers its line1, line2...?

Reply:
@velimd it was previously called continueApplication and isInformationCorrect, I can update it to line1 etc to be more consistent if you'd prefer?

Reply:
na its kl, was just curious hence i still approved.